src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -4,24 +4,51 @@ section: components
 cssPrefix: pf-c-divider
 ---
 
+import './Divider.css'
+
 ## Examples
 ```hbs title=<hr>
-{{#> divider}}{{/divider}}
+{{> divider}}
 ```
 
 ```hbs title=<li>
 <ul>
   <li>List item one</li>
-  {{#> divider divider--type="li"}}{{/divider}}
+  {{> divider divider--type="li"}}
   <li>List item two</li>
 </ul>
 ```
 
 ```hbs title=<div>
-{{#> divider divider--type="div"}}{{/divider}}
+{{> divider divider--type="div"}}
+```
+
+```hbs title=Insets
+<div>No inset</div>
```

Comment:
I was expecting the inset in the example to match the title - not match the previous examples plus what's in the title. Maybe we could update the descriptions to describe all of the insets it uses? I also think it's fine just to have a couple of examples. Maybe a simple one "small inset on sm" then a complex one like "md inset, no inset on md, 3xl inset on lg, lg inset on xl"